src/views/List.jsx

```diff
@@ -1,18 +1,46 @@
 import { ListItem } from '../components';
 import { Navigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
 
 export function List({ data, listToken }) {
+	const [searchValue, setSearchValue] = useState('');
+	const [items, setItems] = useState([]);
+	const [filteredItems, setFilteredItems] = useState([]);
+
+	useEffect(() => {
+		setItems(data);
+	}, [data]);
```

Comment:
I'm not sure you need a new state for `items`, since the prop `data` is already a state that's managed by App.js, also like here within a useEffect, triggering a rerender when a new item is added to the list. You can check it on line 45 of App.js.

Reply:
Yes, we fixed it, It was an oversight on our end, didn't check to see that data was a state in App.js, but it has been resolved now